transformer_sidecar/pyproject.toml

```diff
@@ -23,10 +23,10 @@ optional = true
 [tool.poetry.group.test.dependencies]
 flake8 = "^5.0.4"
 pytest-mock = "^3.10.0"
-codecov = "^2.1.12"
 pytest = "^7.2.0"
 pytest-flask = "^1.2.0"
 coverage = "^6.5.0"
+requests = "^2.28.2"
```

Comment:
Adding here as `requests` was apparently a `codecov` dependency and is required for the `pytest` test suite to pass.